playbooks/tasks/choose-python.yaml

```diff
@@ -0,0 +1,55 @@
+###############################################################################
```

Comment:
Same goes for this playbook as well.

Reply:
I have added some basic flow documentation to the two task files choose-python.yaml and choose-zoau.yaml. Since these files can not be run directly, I placed the bulk of the documentation in host-setup.yaml